scripts/update_appup.escript

```diff
@@ -292,27 +246,21 @@ parse_appup_diffs(Upgrade, OldUpgrade, Downgrade, OldDowngrade) ->
 find_matching_version(VsnOrRegex, PresentChanges) ->
     proplists:get_value(VsnOrRegex, PresentChanges).
 
-find_old_appup_actions(App, PrevVersion) ->
-    {Upgrade0, Downgrade0} =
-        case locate(ebin_current, App, ".appup") of
-            {ok, AppupFile} ->
-                log("Found the previous appup file: ~s~n", [AppupFile]),
-                {_, U, D} = read_appup(AppupFile),
-                {U, D};
+find_base_appup_actions(App, IsExternal, CurrVersion, PrevVersion) ->
+    Locate = case IsExternal orelse force_rebase(App, CurrVersion) of
+                 true -> fun() -> locate_current_rel(App, ".appup.src") end;
+                 false -> fun() -> locate_prev_rel(App, ".appup.src") end
```

Comment:
I didn't quite understand why should we take the `.appup.src` files from the previous release.  Also, this is different from the previous behavior (both `locate(ebin_current, ...)` and `locate(src, ...)` search the files in the current checkout/revision).

Shouldn't we always consider the "current" `appup.src` when calculating the missing actions?

Reply:
We take the prev release as the baseline (the truth), and add new instructions on top.
This allows us to delete the current appup.src and start over with a deterministic result.

@ieQu1 if I understand it correctly, that was the intention?

Reply:
my bad. misunderstood the "Old" appup.
reverted this part.